uaclient/cli.py

```diff
@@ -1038,6 +1038,15 @@ def main(sys_argv=None):
         sys.exit(1)
     args = parser.parse_args(args=cli_arguments)
     cfg = config.UAConfig()
+
+    http_proxy = cfg.cfg.get("http_proxy")
+    https_proxy = cfg.cfg.get("https_proxy")
+
+    if http_proxy or https_proxy:
+        util.configure_web_proxy(
+            http_proxy=http_proxy, https_proxy=https_proxy
+        )
```

Comment:
Also, if we add these as properties on UAConfig, then we could remove the lines above and this could look like:
```suggestion
    util.configure_web_proxy(
        http_proxy=cfg.http_proxy, https_proxy=cfg.https_proxy
    )
```